build/common/installer/scripts/tomlparser-prom-customconfig.rb

```diff
@@ -325,10 +325,10 @@ def populateSettingValuesFromConfigMap(parsedConfig)
             puts "config::Successfully substituted the placeholders in telegraf conf file for custom prometheus scraping"
             #Set environment variables for telemetry in the sidecar container
             if (!@containerType.nil? && @containerType.casecmp(@promSideCar) == 0)
-              file = File.open("telemetry_prom_config_env_var", "w")
+              file = File.open("prom_config_env_var", "w")
               if !file.nil?
                 #Setting array lengths as environment variables for telemetry purposes
```

Comment:
Could you pls update this comment since it is being used for purposes other than telemetry?

Reply:
Done